agent/engine/task_manager.go

```diff
@@ -81,10 +87,17 @@ type containerTransition struct {
 // task's statuses yourself)
 type managedTask struct {
 	*api.Task
-	engine *DockerTaskEngine
-
-	acsMessages    chan acsTransition
-	dockerMessages chan dockerContainerChange
+	engine             *DockerTaskEngine
+	cfg                *config.Config
+	saver              statemanager.Saver
+	credentialsManager credentials.Manager
+	cniClient          ecscni.CNIClient
+	taskStopGroup      *utilsync.SequentialWaitGroup
```

Comment:
nit: should we do the `*WG` variable name tag for consistency here as well?

Reply:
Yes. I forgot about that. Thanks for catching that.